### DISCUSSION TITLE
有没有必要做一个近期在处理的Issue单供检查项使用

### DISCUSSION THREAD

**Discussion by allMagicNB** [Meloong-Git/PCL#3304] [讨论]
打算开个像#1930 的表，并尽量有详有略供快速排查
我在等一个时机，等待一个好的容易记住的Discussion号（

**Comment by iPodtouch6557** [comment]
~比如 #666 对吧~
我在想要不要开个拒绝放弃黑名单

**Reply by allMagicNB** [reply]
> ~比如 #666 对吧~
> 我在想要不要开个拒绝放弃黑名单
> 

那可太多了……

**Reply by iPodtouch6557** [reply]
所以啊 我在想 要是好办我直接交了
另外是**像#1930**不是**想#1930**

**Comment by WForst-Breeze** [comment]
个人感觉没必要，1930是主要归纳了常见的和不好搜的，再整个处理中的列表有点多此一举，直接搜 issue 的时候点那个处理中标签就好了，反而太多额外的页面会很乱，而且可能重复反馈会变得更多，因为交个issue还要搜三遍，多少是可能让人感觉一眼看过去就不想搜

**Reply by allMagicNB** [reply]
> 个人感觉没必要，1930是主要归纳了常见的和不好搜的，再整个处理中的列表有点多此一举，直接搜 issue 的时候点那个处理中标签就好了，反而太多额外的页面会很乱，而且可能重复反馈会变得更多，因为交个issue还要搜三遍，多少是可能让人感觉一眼看过去就不想搜

首先我这是**快速排查**，主要归纳的是近期在处理的反馈
其次，GitHub新手不会怎么限定Label
至于搜三遍……除非是很守规矩的人才会去搜
而且我应该不会做太长，一眼就能看完，没必要搜吧……

**Reply by WForst-Breeze** [reply]
> 除非是很守规矩的人才会去搜

这就出现了问题：守规矩的一去 Issue 页面搜就能搜到，不需要这个表格，不守规矩的不会去搜，不需要这个表格

**Reply by allMagicNB** [reply]
> > 除非是很守规矩的人才会去搜
> 
> 这就出现了问题：守规矩的一去 Issue 页面搜就能搜到，不需要这个表格，不守规矩的不会去搜，不需要这个表格

不一定，Issue标题千变万化，有时候不多想几个关键词还搜不到
